deploy-manage/security/fips-ingest.md

```diff
@@ -0,0 +1,95 @@
+
+
+### APM Server [ingest-apm-limitations]
+
+* The [Secrets Keystore](/solutions/observability/apm/secrets-keystore-for-secure-settings.md) is not supported. 
+
+### Filebeat [ingest-filebeat-limitations]
+
+* The [Secrets Keystore](beats://reference/filebeat/keystore.md) is not supported. 
+* The [Translate GUID processor](beats://reference/filebeat/processor-translate-guid.md) is not supported.
+* The [Fingerprint processor](beats://reference/filebeat/fingerprint.md) does not support the md5 and sha1 method. 
+* The [Community ID Network Flowhash processor](beats://reference/filebeat/community-id.md) is not supported. 
+* The [Azure module](beats://reference/filebeat/filebeat-module-azure.md) including the [Azure eventhub input](beats://reference/filebeat/filebeat-input-azure-eventhub.md) and the [Office 365 module (Beta)](beats://reference/filebeat/filebeat-module-o365.md) is currently not supported. The [Add Cloud Metadata processor](beats://reference/filebeat/add-cloud-metadata.md) does not support the Azure Virtual Machine provider currently. 
+* The [GCP Pub/Sub input](beats://reference/filebeat/filebeat-input-gcp-pubsub.md) is not supported for now. 
+
+### Metricbeat [ingest-metricbeat-limitations]
+
+* The [Secrets Keystore](beats://reference/metricbeat/keystore.md) is not supported. 
+* The [Translate GUID processor](beats://reference/metricbeat/processor-translate-guid.md) is not supported.
+* The [Fingerprint processor](beats://reference/metricbeat/fingerprint.md) does not support the md5 and sha1 method. 
+* The [Community ID Network Flowhash processor](beats://reference/metricbeat/community-id.md) is not supported. 
+* The [Azure module](beats://reference/metricbeat/metricbeat-module-azure.md) is currently not supported. The [Add Cloud Metadata processor](beats://reference/metricbeat/add-cloud-metadata.md) does not support the Azure Virtual Machine provider currently. 
+* The [Beta KVM module](beats://reference/metricbeat/metricbeat-module-kvm.md) is not yet supported.
+* The [Mongo DB module](beats://reference/metricbeat/metricbeat-module-mongodb.md) is not supported. 
+* The [MySQL](beats://reference/metricbeat/metricbeat-module-mysql.md), [PostgreSQL](beats://reference/metricbeat/metricbeat-module-postgresql.md), [MSSQL](beats://reference/metricbeat/metricbeat-module-mssql.md) and [SQL](beats://reference/metricbeat/metricbeat-module-sql.md) modules are not supported. 
+* The [Oracle module](beats://reference/metricbeat/metricbeat-module-oracle.md) is not supported. 
+
+### Elastic Agent and Fleet Server [ingest-limitations-agent]
+
+When you use {{agent}} and {{fleet-server}}, these limitations apply:
+* The [Prometheus Receiver](https://www.elastic.co/docs/reference/integrations/prometheus) is not supported. 
+* Running {{agent}} in [OpenTelemetry mode](https://github.com/elastic/elastic-agent/blob/main/internal/pkg/otel/README.md) is not yet supported. 
```

Comment:
@cmacknz do you have suggestions whether this should be enough to explain that otel is not supported in FIPS mode yet, or should we additionally list `fbreceiver` and `mbreceiver`, and potentially also list them in the filebeat and metricbeat sections?

Reply:
@cmacknz and I discussed this in a sync. 

Given that the OpenTelemetry mode is not supported, receivers and exporters do not need to be particularly mentioned here. 

Let's delete L82 and rewrite L83 as follows:

* Running {{agent}} in [OpenTelemetry mode](https://github.com/elastic/elastic-agent/blob/main/internal/pkg/otel/README.md) is not yet supported. This includes all receivers, such as Filebeat Receiver, Metricbeat Receiver, [Prometheus Receiver](https://www.elastic.co/docs/reference/integrations/prometheus).